ansible/roles/terminal/tasks/terminal.yml

```diff
@@ -9,17 +9,14 @@
   retries: 10
   delay: 2
 
-- name: install termite terminal emulator
+- name: install alacritty terminal emulator
   pacman:
-    name: "{{ item }}"
+    name: "alacritty"
     state: present
   register: task_result
   until: task_result is success
   retries: 10
   delay: 2
-  with_items:
-    - termite
-    - termite-terminfo
```

Comment:
Oh noes, I need alacritty in my life. We could change this task to install a List of desired terminal emulators as well as their terminfo packages?